basics/loops.md

```diff
@@ -260,9 +285,30 @@ c
 >>>
 ```
 
+You can print the multiplication tables.
+
+```python
+>>> table = 2
+>>> for i in range(1, 11):
+...     print(table,"x", i, "=",(table*i))
+...
+2 x 1 = 2
+2 x 2 = 4
+2 x 3 = 6
+2 x 4 = 8
+2 x 5 = 10
+2 x 6 = 12
+2 x 7 = 14
+2 x 8 = 16
+2 x 9 = 18
+2 x 10 = 20
+>>>
+```
+
```

Comment:
Is there an `## Examples` section on this page? This could go there. If there isn't, you can create one.